sys/hoon.hoon

```diff
@@ -821,10 +821,13 @@
 ++  swag                                                ::  slice
   |*  {{a/@ b/@} c/(list)}
   (scag +<-> (slag +<-< c))
+::  +turn: transform each value of list :a using the function :b
 ::
-++  turn                                                ::  transform
+++  turn
   ~/  %turn
-  |*  {a/(list) b/gate}
+  |*  [a=(list) b=gate]
+  =>  .(a (homo a))
+  ^-  (list _(b ?~(a !! i.a)))
```

Comment:
I think I prefer `_?>(?=(^ a) (b i.a))`, and I think that's the convention for these casts. Now that I write it out, I'm surprised to see yours is shorter.

Reply:
I too have a minor preference for `_?>(?= ...)` for consistency, but this is only a minor preference.